app/views/step_by_step_pages/unpublish.html.erb

```diff
@@ -23,36 +23,32 @@
   action: 'Unpublish step by step'
 %>
 
-<div class="callout callout-warning add-bottom-margin">
-  <div class="callout-title">
-    Warning
-  </div>
-  <div class="callout-body">
-    Unpublish a page from GOV.UK. This can’t be undone.
-  </div>
-</div>
+<%= render "govuk_publishing_components/components/notice", {
+  title: "Warning",
+  description_govspeak: "You are unpublishing a page from GOV.UK. This can’t be undone."
+} %>
 
-<div class="row">
-  <div class="col-md-7">
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
     <%= form_tag(step_by_step_page_unpublish_path(@step_by_step_page), method: :post) do %>
-      <h3 class="remove-top-margin add-bottom-margin">Unpublish</h3>
+      <h3 class="govuk-heading-m">Unpublish</h3>
 
-      <div class="form-group">
-        <label for="redirect_url">Redirect to</label>
-        <span class="normal">
-          For example: <code>/redirect-to-replacement-page</code>
-        </span>
-        <%= text_field_tag :redirect_url %>
-      </div>
+      <%= render "govuk_publishing_components/components/input", {
+        label: {
+          text: "Redirect to"
+        },
+        name: :redirect_url,
+        hint: sanitize("For example: <code>/redirect-to-replacement-page</code>")
+      } %>
 
       <div class="form-group">
-        <%= button_tag 'Unpublish',
+        <%= button_tag 'Unpublish step by step',
```

Comment:
Can this be switched to use a button component?

Reply:
Done